### ISSUE TITLE
Editor a11y usability testing and validation 

### ISSUE THREAD

**Issue by EWashb** [department-of-veterans-affairs/va.gov-cms#8526]
The Claro theme for the Drupal CMS was launched in Q1 2022. We have a need to validate that both the out-of-the-box interface and MVP design decisions are accessible. We should be especially mindful of users with screen readers. 

As a user, I need to have neutral or positive feelings about the new Drupal CMS theme so I can easily and quickly update my content. 

### Affected users and stakeholders

* CMS editors across product

Related to #1352 

### Hypothesis

We believe that compared to the previous theme, the Claro launch has provided a neutral or better experience for users. 

## Assumptions
* We assume that the Claro theme has provided a neutral or better experience.
* We assume that there will still be usability painpoints not addressed by the Claro theme

## Acceptance Criteria
- [ ] Attempt to recruit participant that uses a screen reader
- [ ] Usability testing completed
- [ ] Synthesize and identify recommendations for next steps
- [ ] Readout conducted

## Design principles

Veteran-centered
- [x] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [x] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [x] `Consistent`: Reduce users’ mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team
Please check the team(s) that will do this work. 

- [ ] `CMS Program`
- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team ` (leave Sitewide unchecked and check the specific team instead)

**Comment by EWashb** [comment]
We will need to reevaluate our approach as we onboard the Sitewide Crew Accessibility Lead.

**Comment by EWashb** [comment]
A lot of this work has been done in-house with our accessibility lead. Because of their work on auditing the CMS this will not be needed. Closing
